src/databaseCheck.js

```diff
@@ -0,0 +1,67 @@
+const { pool, userPool } = require('./db');
+const LOG = require('./utils/Log');
```

Comment:
You need use the values from BACKSTAGE_DB_NAME, BACKSTAGE_DB_USER, BACKSTAGE_DB_PWD, BACKSTAGE_DB_HOST here